Sources/NSTextCheckResult+NamedCapture.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  NSTextCheckResult+NamedCapture.swift
```

Comment:
I suppose the file name should be renamed as _NSTextCheckingResult+NamedCaptureGroup_.swift. Mouthful, but the consistency would be guaranteed.